dbms/src/Core/SettingsCommon.cpp

```diff
@@ -63,6 +63,17 @@ void SettingNumber<Type>::set(const String & x)
     set(parse<Type>(x));
```

Comment:
Maybe it will be useful to make "accurate" version of `parse()`. I mean, we have function `accurate::convertNumeric()` which throws an exception if it can't convert correctly, why not have function `accurate::parse<T>()` which might be a bit slower than `DB::parse<T>()`, but more accurate.

Reply:
Yes, maybe some small separate file.